.github/workflows/ci-kickoff-manual.yml

```diff
@@ -13,6 +13,33 @@ on:
         default: ''
 
 jobs:
+  get-pr-base-ref:
```

Comment:
out of curiosity, why is this necessary?

Reply:
Such that, in the `didc_checks` job of the called `ci-pr-only` workflow, we can calculate the merge-base between `commit-sha` and the base branch of the PR.